test/runtests.py

```diff
@@ -197,10 +236,11 @@ def _process_msg(self, msg):
             'Failed' if fail else 'Passed',
             self._short_name(filename))
         padding = self._last_len - len(line)
+        log_message(line)
```

Comment:
nit: Move this `log_message` down one line, so that we always follow `print and log` order, and let `padding` calculation go together with consumption in print.